aspnetcore-features/HowToPreventAHangfireJobFromRunning/Tests/Job2IntegrationTest.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace Tests;
+
+public class Job2IntegrationTests : IClassFixture<ApiApplicationFactory>
+{
+    private readonly string _baseUri = "http://localhost:5000/api/jobs/";
+
+    private readonly HttpClient _client;
+
+    public Job2IntegrationTests(ApiApplicationFactory factory)
+    {
+        _client = factory.CreateClient();
+    }
+
+    [Fact]
+    public async Task GivenJobsController_WhenCreatingManyOccurrencesOfAGivenJobButUsingSkipConcurrentExecutionAttribute_ThenTheExtraJobsAreBeingSkipped()
```

Comment:
Function name is too big, try to shorten it

Reply:
Done

Reply:
Reviewed